dynawo/sources/Common/DYNTrace.cpp

```diff
@@ -148,19 +150,32 @@ void Trace::addAppenders_(const std::vector<TraceAppender>& appenders) {
   logging::attributes::current_thread_id::value_type currentId =
     logging::attributes::current_thread_id().get_value().extract<logging::attributes::current_thread_id::value_type>().get();
 
+  for (const TraceAppender& appender : appenders) {
+    if (appender.getTag() == Trace::variables()) {
+      variablesAppenderAndThreadId_ = std::make_pair(appender, currentId);
+      // set VARIABLES tagged appender to overwrite the VARIABLES log file
+      variablesAppenderAndThreadId_.get().first.setOpenMode(std::ios_base::out | std::ios_base::trunc);
```

Comment:
Why is this needed? If we can avoid specific sink modification it would be great